src/widget/file/filepicker.js

```diff
@@ -241,6 +242,15 @@ class Filepicker extends Widget {
         }
     }
 
+    _resizeImage( file, mediaType ) {
+        if ( mediaType !== 'image/*' ) {
```

Comment:
maybe also add a check to see if this.props.maxPixels is defined (to be ready for future)

Reply:
Checking added.